test/tests/models/tx/Test_Input.cpp

```diff
@@ -1 +1,93 @@
-// TODO: Test Serialization/Deserialization (Raw & JSON)
\ No newline at end of file
+#include <catch.hpp>
+
+#include <mw/crypto/Crypto.h>
+#include <mw/crypto/Random.h>
+#include <mw/models/tx/Input.h>
+
+TEST_CASE("Plain Tx Input")
+{
+    Commitment commit(Random::CSPRNG<33>().GetBigInt());
```

Comment:
If you switched this to use Commitment::FromHex instead, then you could assert that the serialized input is an exact byte array, which seems more ideal than just recreating the serialization logic in the test.

This is probably good for now though. We can just make a note to go back and simplify this later.